functions/package.json

```diff
@@ -10,8 +10,11 @@
   },
   "dependencies": {
     "@sendgrid/mail": "^6.3.1",
-    "firebase-admin": "~6.0.0",
-    "firebase-functions": "^2.1.0"
+    "firebase-admin": "^8.2.0",
+    "firebase-functions": "^3.0.1"
   },
-  "private": true
+  "private": true,
+  "engines": {
+    "node": "8"
```

Comment:
Don't know what version of node was used in practice before, but got it working with 8 and made it explicit to help the next person.